assets/js/googlesitekit/widgets/components/widget-area.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WidgetArea component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { string } from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../datastore';
+import Widget from './widget';
+
+const { useSelect } = Data;
+
+const WidgetArea = ( { area } ) => {
+	const widgetArea = useSelect( ( select ) => select( STORE_NAME ).getWidgetArea( area ) );
+	const widgets = useSelect( ( select ) => select( STORE_NAME ).getWidgets( area ) );
+
+	return (
+		<div className={ classnames( 'mdc-layout-grid', 'WidgetArea', `WidgetArea-${ widgetArea.style }`, `WidgetArea-${ widgetArea.slug }` ) }>
```

Comment:
The readability of separating out each className, especially when some are dynamic, is worth using `classnames` here I think. Otherwise they're a big string and it's a bit more unwieldy, especially for things like ESLint auto-formatting.